### ISSUE TITLE
Examples missing

### ISSUE THREAD

**Issue by PeterHuewe** [ARM-software/psa-api#2]
Hi,
thanks for the publication of psa-api.
Unfortunately, the mentioned example section is still empty.

**Comment by ndevillard** [comment]
Correct, we still have some code to release to provide examples. Just curious: would you be more interested in examples of PSA-API-providing implementations, or PSA-API-using ones?

**Comment by kriskwiatkowski** [comment]
Personally, I find those examples useful:
https://www.silabs.com/documents/public/application-notes/an1311-mbedtls-psa-crypto-porting-guide.pdf

**Comment by hannestschofenig** [comment]
Here are examples: https://github.com/Mbed-TLS/mbedtls/pull/5064
